src/dune/package.ml

```diff
@@ -398,6 +398,7 @@ let decode ~dir =
      and+ name = field "name" Name.decode
      and+ synopsis = field_o "synopsis" string
      and+ description = field_o "description" string
+     and+ version = field_o "version" string
```

Comment:
Could you make sure the field is only available starting 2.5? There should be `since` calls elsewhere in this file (or in `dune_project.ml`, `dune_file.ml`) for an example on how to do it.

Reply:
I think this is done!

Reply:
@rgrinberg it looks like my changes:
```ocaml
     and+ version =
       field_o "version" (Dune_lang.Syntax.since Stanza.syntax (2, 5) >>> string)
```
Are too strict and prevent backward-compatible top-level `version` field.. Any pointer on how to only to the check for the package-specific field? Thks!

Reply:
Or perhaps it should be a more explicit name like `version_override`. I'll go with that!